.devops/azure-templates/chart-current-version.yml

```
@@ -0,0 +1,23 @@
+# Read the current chart version and output it two variables
+# -> 'chart_current_version.version' : the Chart version
+# -> 'chart_current_version.appVersion': the application version
+
+steps:
+  - task: Bash@3
+    name: chart_current_version
+    displayName: 'Read chart current version'
+    inputs:
+      targetType: "inline"
+      script: |
+        CHART_FILE="helm/Chart.yaml"
+        version=$(yq -r '.version' $CHART_FILE)
+        appVersion=$(yq -r '.appVersion' $CHART_FILE)
+        trimmed_repo_url=$(echo $(Build.Repository.Uri) | sed -e "s/\.git//g")
+        releaseUrl=$(echo $trimmed_repo_url/releases/tag/$appVersion)
```

```suggestion
        releaseUrl=$(echo $trimmed_repo_url/releases/tag/$version)
```

The version used for the github releases is the chart one, use this instead of app version here